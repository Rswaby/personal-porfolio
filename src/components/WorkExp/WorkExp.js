import React from 'react'
import {
    Grid,
    Image,
    Header,
    Container,
} from 'semantic-ui-react'

const WorkExp = (props) => (
    
    <Grid container>
        <Grid.Column width={4}>
            <Image size='small' src={props.work.image} />
        </Grid.Column>
        <Grid.Column width={9} centered>
            <Header as="h3" textAlign="left">{props.work.company}</Header>
            <Container text fluid textAlign="left">
                <p>{props.work.description}</p>
            </Container>
        </Grid.Column>
        <Grid.Column width={3}>
        </Grid.Column>
    </Grid>

)

export default WorkExp