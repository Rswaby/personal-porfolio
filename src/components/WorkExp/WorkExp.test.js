
/* eslint-disable */
import React from 'react';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { mount } from 'enzyme';
import WorkExp from './WorkExp'
import renderer from 'react-test-renderer';

enzyme.configure({ adapter: new Adapter() });



describe('<Navbar /> ', () => {
    const work = {
        "company":"Bright Power",
        "image":"https://pbs.twimg.com/profile_images/634075551964106752/pBN6np7z_400x400.jpg",
        "title":"Software Developer Intern",
        "years":"Jan 2017 - August 2017",
        "stack": [
          {"name":"Java"},
          {"name":"Python"},
          {"name":"Spring boot"},
          {"name":"testing: JUnit, Mockito"}
        ],
        "description":"During My Time at Bright Power some of my Responsibilities including Writing unit testing for Spring MVC application for REST controllers using JUnit, Mockito. I also Created test suite for newly created API services using postman to examine performance. I Conducted other QA responsibilities such as regression/feature testing in order to identify Defects in pre-released software. One project I worked on creating and maintaining a data scraping tool using selenium and beautifulsoup for internal use."
      }
    it('should render correctly', () => {
        const mock = jest.fn();
        const wrapper = mount(<WorkExp work={work} />);
    });
    test('<Navbar/> snapshot', () => {
        ///
        const component = renderer.create(<WorkExp work={work} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })

})
