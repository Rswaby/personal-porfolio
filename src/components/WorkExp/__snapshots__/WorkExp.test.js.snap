// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar />  <Navbar/> snapshot 1`] = `
<div
  className="ui container grid"
  style={
    Object {
      "marginBottom": "3em",
    }
  }
>
  <div
    className="three wide column"
  >
    <img
      className="ui small image"
      src="https://pbs.twimg.com/profile_images/634075551964106752/pBN6np7z_400x400.jpg"
    />
  </div>
  <div
    centered={true}
    className="nine wide column"
  >
    <div
      className="ui message"
      style={
        Object {
          "height": "220px",
        }
      }
    >
      <h3
        className="ui left aligned header"
      >
        Bright Power
      </h3>
      <hr />
      <div
        className="ui text fluid left aligned container"
      >
        <p>
          During My Time at Bright Power some of my Responsibilities including Writing unit testing for Spring MVC application for REST controllers using JUnit, Mockito. I also Created test suite for newly created API services using postman to examine performance. I Conducted other QA responsibilities such as regression/feature testing in order to identify Defects in pre-released software. One project I worked on creating and maintaining a data scraping tool using selenium and beautifulsoup for internal use.
        </p>
      </div>
    </div>
  </div>
  <div
    className="four wide column"
  >
    <div
      className="ui message"
      style={
        Object {
          "height": "220px",
        }
      }
    >
      <h4
        className="sub header"
        textAlign="left"
      >
         
        <p>
          Role
        </p>
      </h4>
      <hr />
      <p>
        Software Developer Intern
      </p>
      <h5
        className="ui header"
      >
        <p>
          Technologies used:
        </p>
      </h5>
    </div>
  </div>
</div>
`;
